testcases/PowerNVDump.py

```diff
@@ -912,7 +917,7 @@ def setup_san(self):
         if self.distro == "rhel":
             self.c.run_command("sfdisk --delete %s" % self.dev_path)
             self.c.run_command("echo , | sfdisk --force %s" % self.dev_path, timeout=120)
-            try: self.c.run_command("umount %s1" % self.dev_path)
+            try: self.c.run_command("umount /var/crash")
```

Comment:
@abdhaleegit the script trying to mount the given dev.path to /var/crash. 
so it would be good umount the "/var/crash" if its already mounted instead of dev.path itself.

Reply:
But this will change the default behaviour and may break regular crash dump case where a custom crash directory has been provided. If absolutely required can you restrict the change to watchdog case 
if (watchdog) varcrash else dev_path